app/jobs/regular/bulk_user_title_update.rb

```diff
@@ -18,6 +19,23 @@ def execute(args)
       end
     end
 
+    # Not called from the job because this needs to happen during the destroy request
+    def self.clear_titles_for_granted_badge(granted_badge_id)
+      ActiveRecord::Base.transaction do
+        DB.exec(<<~SQL, granted_title_badge_id: granted_badge_id, title: '', updated_at: Time.now)
+          UPDATE users AS u
+          SET title = :title, updated_at = :updated_at
```

Comment:
Nitpick, but why pass in `title` if you are just setting it to `''`? `SET title = ''`